tests/test_observatory.py

```diff
@@ -205,6 +205,12 @@ def test_from_yaml(bm):
         Observatory.from_yaml(3)
 
 
+def test_from_ska():
+    Observatory.from_ska(subarray_type="AA*", array_type="low", frequency=300.0 * units.MHz)
+    Observatory.from_ska(subarray_type="AA*", array_type="mid", frequency=300.0 * units.MHz)
+    Observatory.from_ska(subarray_type="AA4", array_type="low", frequency=300.0 * units.MHz)
```

Comment:
let's make sure that the observatory has the right number of baselines etc. as well